### PR TITLE
Calendar

### PR DESCRIPTION
**Pull request by RebeccaStankus**
https://highfidelity.atlassian.net/browse/BUGZ-487

### CODE REVIEW

File: usefulUtilities/hiFiCalendar/entities/calendarEntities.json [hunk 1]
```diff
@@ -48,54 +50,55 @@
     },
     "damping": 0,
     "angularDamping": 0,
-    "collisionless": true,
```
Comment (zfox23) [inline usefulUtilities/hiFiCalendar/entities/calendarEntities.json]: why is this entity no longer collisionless?

File: usefulUtilities/hiFiCalendar/entities/calendarEntities.json [hunk 2]
```diff
@@ -2,34 +2,36 @@
...
+    "id": "{455f56c8-9a76-420f-a8ab-6dad5da7358a}",
     "type": "Text",
-    "lastEdited": 1559239932550773,
-    "parentID": "{a81d9ae5-6df9-4d2d-9d40-4062752f1973}",
-    "name": "Calendar_MR1_Main_RoomColorOccupant",
+    "lastEdited": 1561571024131146,
+    "parentID": "{2175cd3b-9bfd-4892-b666-98d50a975f6d}",
+    "name": "Calendar_MR1_Main_RoomOccupantsList",
+    "userData": "{\"roomOccupantsListID\":\"\",\"meetingZoneID\":\"\"}",
+    "description": "CFGBLE -",
```
Comment (zfox23) [inline usefulUtilities/hiFiCalendar/entities/calendarEntities.json]: is this actually configurable?

File: usefulUtilities/hiFiCalendar/entities/calendarEntities.json [hunk 3]
```diff
@@ -48,54 +50,55 @@
...
-    "id": "{c2894bb0-eb00-4c2e-b5c7-8776b7eb2ffd}",
+    "id": "{8134ccd7-2083-48ee-8e3c-28d45d2911ec}",
     "type": "Text",
-    "lastEdited": 1559239932550661,
-    "parentID": "{e5818ba9-1e52-4617-bd87-a1bc9bd2fe77}",
-    "name": "Calendar_MR1_Main_RoomColor",
+    "lastEdited": 1561571056513527,
+    "parentID": "{2175cd3b-9bfd-4892-b666-98d50a975f6d}",
+    "name": "Calendar_MR1_Main_RoomColorOccupant",
+    "description": "CFGBLE -",
```
Comment (zfox23) [inline usefulUtilities/hiFiCalendar/entities/calendarEntities.json]: is this actually configurable?

File: usefulUtilities/hiFiCalendar/entities/calendarEntities.json [hunk 4]
```diff
@@ -183,7 +184,7 @@
         }
     },
     "serverScripts": "http://hifi-content.s3-us-west-1.amazonaws.com/Experiences/Releases/usefulUtilities/clock/v1.0/entityResources/clockServer.js",
-    "text": "",
+    "text": "10:44am PDT",
```
Comment (zfox23) [inline usefulUtilities/hiFiCalendar/entities/calendarEntities.json]: please remove this text

File: usefulUtilities/hiFiCalendar/entities/calendarEntities.json [hunk 5]
```diff
@@ -198,34 +199,35 @@
...
-    "id": "{a81d9ae5-6df9-4d2d-9d40-4062752f1973}",
+    "id": "{c2894bb0-eb00-4c2e-b5c7-8776b7eb2ffd}",
     "type": "Text",
-    "lastEdited": 1559239905855894,
-    "name": "Calendar_MR1_Main_RoomOccupantsList",
-    "userData": "",
+    "lastEdited": 1561571056513416,
+    "parentID": "{2175cd3b-9bfd-4892-b666-98d50a975f6d}",
+    "name": "Calendar_MR1_Main_RoomColor",
+    "description": "CFGBLE -",
```
Comment (zfox23) [inline usefulUtilities/hiFiCalendar/entities/calendarEntities.json]: is this actually configurable?

File: usefulUtilities/hiFiCalendar/entities/calendarEntities.json (hunk 2 above)
Comment (MiladNazeri) [reply usefulUtilities/hiFiCalendar/entities/calendarEntities.json]: This one needs the id of the meeting zone.  Does that qualify it for the configurable label?

File: usefulUtilities/hiFiCalendar/entities/calendarEntities.json (hunk 3 above)
Comment (MiladNazeri) [reply usefulUtilities/hiFiCalendar/entities/calendarEntities.json]: Not on this one

File: usefulUtilities/hiFiCalendar/entities/calendarEntities.json (hunk 2 above)
Comment (zfox23) [reply usefulUtilities/hiFiCalendar/entities/calendarEntities.json]: yes

File: usefulUtilities/hiFiCalendar/entities/calendarEntities.json (hunk 5 above)
Comment (MiladNazeri) [reply usefulUtilities/hiFiCalendar/entities/calendarEntities.json]: No one this one

File: usefulUtilities/hiFiCalendar/entities/calendarEntities.json [hunk 6]
```diff
@@ -313,8 +320,8 @@
     "angularDamping": 0,
     "collisionMask": 23,
     "collidesWith": "static,dynamic,kinematic,otherAvatar,",
-    "serverScripts": "https://hifi-content.s3.amazonaws.com/Experiences/Releases/usefulUtilities/hiFiCalendar/2019-05-23_14-43-38/entityServerScripts/roomScheduleServer.js",
-    "text": "",
+    "serverScripts": "https://hifi-content.s3.amazonaws.com/Experiences/Releases/usefulUtilities/hiFiCalendar/2019-06-14_07-23-00/entityServerScripts/roomScheduleServer.js",
+    "text": "Daily LilyPad Bug Triage\n11:00 am - 12:00 pm PDT\n\nWeekly PM/Product Sync\n1:00 pm - 2:00 pm PDT",
```
Comment (MiladNazeri) [inline usefulUtilities/hiFiCalendar/entities/calendarEntities.json]: scrub

File: usefulUtilities/hiFiCalendar/entities/calendarEntities.json (hunk 1 above)
Comment (RebeccaStankus) [reply usefulUtilities/hiFiCalendar/entities/calendarEntities.json]: That is not my change. I pulled this from the www content but will add the collisionless property.

**Pull request closed without merging**